.github/workflows/publish.yml

```diff
@@ -6,20 +6,42 @@ jobs:
   deploy-pypi:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
-    - name: Set up Python
-      uses: actions/setup-python@v2
-      with:
-        python-version: 3.9
-    - name: Install dependencies
-      run: |
-        python -m pip install --upgrade pip
-        pip install -r local-requirements.txt
-        pip install -e .
-        python setup.py bdist_wheel --all
-        python -m playwright install-deps
-    - name: Publish package
-      env:
-        TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
-        TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
-      run: twine upload dist/*
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r local-requirements.txt
+          pip install -e .
+          python setup.py bdist_wheel --all
+          python -m playwright install-deps
+      - name: Publish package
+        env:
+          TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
+          TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
+        run: twine upload dist/*
+
+  deploy-conda:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Get conda
+        uses: conda-incubator/setup-miniconda@v2
+        with:
```

Comment:
lets specify the Python version there too? 3.8 since 3.9 has some bugs in combination conda build.
e.g. https://github.com/Anaconda-Platform/anaconda-client/issues/555

Reply:
I tried it and it worked without even so I didn't added that, would add in next commit

Reply:
Locally it failed uploading for me yesterday silently without any errors which took me a hour to figure out. So I expect it also failing uploading in the CI. :/ I want to avoid recreating tags etc

Talking about releasing, we can extend the deployment GitHub workflow by a custom event trigger so we can trigger it manually later on the release branch to upload v1.12 from GHA directly: https://docs.github.com/en/actions/reference/events-that-trigger-workflows#workflow_dispatch